WalletWasabi.Fluent/Views/Wallets/Home/History/HistoryTablePlaceholder.axaml

```diff
@@ -0,0 +1,41 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:history="clr-namespace:WalletWasabi.Fluent.ViewModels.Wallets.Home.History"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:DataType="history:HistoryViewModel"
+             x:CompileBindings="True"
+             x:Class="WalletWasabi.Fluent.Views.Wallets.Home.History.HistoryTablePlaceholder">
+  <Border ClipToBounds="False"
+          CornerRadius="5"
+          Margin="6"
+          Padding="16"
+          Background="{DynamicResource TileRegionColor}"
+          BoxShadow="{DynamicResource InfoBoxShadow}">
+    <DockPanel HorizontalAlignment="Center" VerticalAlignment="Center">
+      <Label Content="Recent transactions will appear here once you make some transactions or when your wallet finishes loading."
```

Comment:
Maybe we can remove the duplicate word?
```suggestion
      <Label Content="Recent transactions will appear here once you make some or when your wallet finishes loading."
```

Reply:
That one is hard, maybe we can say:
`Recent transactions will appear here after they are created or when your wallet finishes loading.`

Reply:
That is a bit better.
Shouldnt we also remove the word `Recent`? because we display all the transactions right?

Reply:
Agree, pushed fixed wording.